misk/src/main/kotlin/misk/web/dashboard/WebTabResourceModule.kt

```diff
@@ -26,24 +27,44 @@ import misk.web.resources.StaticResourceEntry
  * @property resourcePath JVM path for non-Development environment static resources (includes `classpath:/` prefix)
  */
 class WebTabResourceModule(
-  val environment: Environment,
+  private val isDevelopment: Boolean,
   val slug: String,
   val web_proxy_url: String,
   val url_path_prefix: String = "/_tab/$slug/",
   val resourcePath: String = "classpath:/web/_tab/$slug/"
 ) : KAbstractModule() {
+
+  constructor(
```

Comment:
Thanks for these changes and maintaining the existing constructor with deprecation